### PR TITLE
bump docker version

### LINKED ISSUE #168: Upgrade docker version
Upgrade docker version

### PR DESCRIPTION
**Pull request by dylanhitt**
Fixes #168 

Also opening an issue to cancel [`ContainerWait`](https://github.com/commander-cli/commander/blob/bump-docker-version/pkg/runtime/docker_executor.go#L103) with a timeout. Similar to the [example](https://pkg.go.dev/github.com/docker/docker/client#Client.ContainerWait). This should be configurable when configuring a docker node.

## Checklist

 - [ ] Added unit / integration tests for windows, macOS and Linux? 
 - [ ] Added a changelog entry in [CHANGELOG.md](https://github.com/commander-cli/commander/blob/master/CHANGELOG.md)?
 - [ ] Updated the documentation ([README.md](https://github.com/commander-cli/commander/blob/master/README.md), [docs](https://github.com/commander-cli/commander/blob/master/docs))?
 - [ ] Does your change work on `Linux`, `Windows` and `macOS`?

### CODE REVIEW

**Comment by dylanhitt** [comment]
Need to fix some issues found here https://app.circleci.com/pipelines/github/commander-cli/commander/115/workflows/a77c0731-a3fd-4b1b-843c-4034d7db508e/jobs/423?invite=true#step-104-936.

Also it seems this error is present on master not sure it's ever passed. https://app.circleci.com/pipelines/github/commander-cli/commander/93/workflows/86347d7e-880e-470a-a16a-b9519a92bf74/jobs/354?invite=true#step-104-893

File: Makefile [hunk 1]
```diff
@@ -75,7 +75,7 @@ integration-unix: build
 
 .PHONY: integration-linux
 integration-linux: build
-	$(info INFO: Starting build $@)
```
Comment (SimonBaeumer) [inline Makefile:78]: Is this a mistake? 🤔

Comment (dylanhitt) [reply Makefile:78]: ah yeah, probably through my hours of head banging the DNS issues. 

Kinda want to redo the integration test to perform the ssh and docker test as separate jobs, so this kinda got put on the back burner.

Comment (dylanhitt) [reply Makefile:78]: Also expect a GitHub action example..

**Pull request merged**